### ISSUE TITLE
License Changing - MIT => Apache 2.0

### ISSUE THREAD

**Issue by goyney** [Templarian/MaterialDesign#5480]
**TL;DR:** The Material Design Icons collection, including [@mdi/js](https://www.npmjs.com/package/@mdi/js), [@mdi/svg](https://www.npmjs.com/package/@mdi/svg), [@mdi/font](https://www.npmjs.com/package/@mdi/font), and other distributions of icons and code tooling is being relicensed under Apache 2.0 for icon glyphs and fonts, and MIT for any non-icon/font related assets beginning in the v5.6 release.

---

## The Problem

Material Design Icons is community-led. Anyone who wants to design and contribute an icon to our ever growing pack simply need make an icon following [Material Design guidelines](https://material.io/design/iconography/system-icons.html#design-principles) and [open an issue](https://github.com/Templarian/MaterialDesign/issues/new/choose). Because of this communal nature, we want these community icons to be released in an open way. The MIT license is an obvious choice, and our first pick, for how we want to release these icons. However, we also include official Google-created icons in our pack, being that they invented the Material Design system. Until 2017, Google released their icons under the Create Commons license, which worked well with our original MIT license. However, they have since switched to Apache 2.0, which is still an open license, but is slightly more restrictive than MIT. At this point, our community-led collection entered this weird state of dual-licensure.

## The Solution

To remedy the problem, and with the help of some recent legal advise, we have decided to match Google's license and move Material Design Icons to the Apache 2.0 license. This makes it abundantly clear which license the Material Design Icon collections are released under and removes confusion about which icon glyphs are under which license. Beginning in the v5.6 release of the MDI collections, they will ship with the new Apache 2.0 license.

## Proposed License

```
This icon collection is released as free, open source, and GPL friendly.
You may use it for commercial projects, open source projects, or anything really.

# Icons: Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0)
Some of the icons are redistributed under the Apache 2.0 license. All other
icons are either redistributed under their respective licenses or are
distributed under the Apache 2.0 license.

# Fonts: Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0)
All web and desktop fonts are distributed under the Apache 2.0 license. Web
and desktop fonts contain some icons that are redistributed under the Apache
2.0 license. All other icons are either redistributed under their respective
licenses or are distributed under the Apache 2.0 license.

# Code: MIT (https://opensource.org/licenses/MIT)
The MIT license applies to all non-font and non-icon files.
```

---

## To Do:

- [x] Add LICENSE - https://github.com/Templarian/MaterialDesign-Font/pull/2
- [x] Update LICENSE - https://github.com/Templarian/MaterialDesign/pull/5483
- [x] Update LICENSE and `package.json` - https://github.com/Templarian/MaterialDesign-Webfont/pull/72
- [x] Update LICENSE and `package.json` - https://github.com/Templarian/MaterialDesign-Angular-Material/pull/9
- [x] Update LICENSE and `package.json` - https://github.com/Templarian/MaterialDesign-JS/pull/10
- [x] Update LICENSE and `package.json` - https://github.com/Templarian/MaterialDesign-SVG/pull/13

**Comment by Templarian** [comment]
These license updates are now merged as we ready the v5.6.55 release for this week (which can be [previewed here](https://cdn.materialdesignicons.com/5.6.55/)).

**Comment by Templarian** [comment]
Closing this as 5.6.55 was released the other day.
